types/appointment.d.ts

```diff
@@ -0,0 +1,74 @@
+declare module 'skedify-sdk' {
+  export type AppointmentState =
+    | 'incoming'
+    | 'outgoing'
+    | 'completed'
+    | 'cancelled'
+    | 'accepted'
+    | 'occurred'
+
+  export interface Appointment extends Omit<BaseFields, 'enterprise_id'> {
+    state: AppointmentState;
+    contact_id: string;
+    customer_id: string;
+    subject_id: string;
+    office_id: string;
+    meeting_type: string;
+    first_possibility_id: string;
+    notes?: string;
+    outcome_remark?: string;
+    trailing_buffer_time: number;
+    metadata: any[];
+    customer_showed: boolean;
+    is_success: boolean;
+    video_url_for_contact: string;
+    video_url_for_customer: string;
+    video_test_url_for_contact: string;
+    video_test_url_for_customer: string;
+    cancellation_reason_id?: string;
+    cancelled_by_id?: string;
+    cancelled_by_type?: string;
+    initiated_from: string;
+    initiated_by_id: string;
+    initiated_by_type: string;
+    updated_at: string;
+    updated_by_id: string;
+    updated_by_type: string;
+  }
+
+  export interface Possibility
+    extends Omit<BaseFields, 'enterprise_id' | 'external_id'> {
+    appointment_id: number;
+    start: string;
+    end: string;
+    timezone: string;
+    duration: number;
+    is_accepted: boolean;
+    next_possibility_id: null;
```

Comment:
Q: According to the API this value is always an integer. Shouldn't this reflect this as well?

Reply:
I haven't had a case where I had an actual next possibility, but will add `number | null`